x-pack/plugins/security_solution/public/common/components/header_actions/actions.tsx

```diff
@@ -220,6 +212,11 @@ const ActionsComponent: React.FC<ActionProps> = ({
     onEventDetailsPanelOpened();
   }, [activeStep, incrementStep, isTourAnchor, isTourShown, onEventDetailsPanelOpened]);
 
+  const noteIds = useMemo(
+    () => (eventIdToNoteIds ? eventIdToNoteIds[eventId] || emptyNotes : emptyNotes),
```

Comment:
nit: but could we simplify this to just `eventIdToNoteIds?.[enventId] || emptyNotes`?

Reply:
done here: 801c31448a2